src/components/tasks/card/SuggestionBox.tsx

```diff
@@ -0,0 +1,107 @@
+import { FC, useEffect, useState } from 'react';
+import { DUMMY_PROFILE as placeholderImageURL } from '@/constants/display-sections';
+import classes from '@/components/tasks/card/suggestion.module.scss';
+import fetch from '@/helperFunctions/fetch';
+import { userDataType } from '@/interfaces/user.type';
+
+type Props = {
+    assigneeName: string;
+    showSuggestion: boolean;
+    onClickName: (userName: string) => void;
+};
+
+type GithubInfo = {
+    github_id: string;
+    url: string;
+};
```

Comment:
don't you think these types should be included in a separate files and exported into suggestionBox.tsx and then used

Reply:
I had thought of that, but then i assumed i only need this particular type only for this particular component so i thought why not localize it.

Reply:
please make a different file for it and then import those types in the file and use it that would make your component look nice and optimized.

Reply:
Ok

Reply:
resolved @Pratiyushkumar